### ISSUE TITLE
Chrome Notification asking to save password

### ISSUE THREAD

**Issue by MikeZeDev** [manga-download/haruneko#100]
* Porting a connector to Haru
* Ask to update manga list
* Got this 
![image](https://user-images.githubusercontent.com/10671518/220426575-75fb2f9f-c4e9-4347-9b3c-6592dc82048b.png)

👀

**Comment by Sheepux** [comment]
A connector with custom logins in the settings i guess ?

**Comment by MikeZeDev** [comment]
Nope, there is no such things for now. Thats why its weird. And its hard to replicate.

**Comment by MikeZeDev** [comment]
Got it on splash page too

**Comment by Pierre-Monier** [comment]
[This](https://github.com/nwjs/nw.js/issues/5944) might be helpful

**Comment by Sheepux** [comment]
Issue is going stale, nobody reported it again I think.
I'm voting in favor of closing
